show.js

```diff
@@ -0,0 +1,67 @@
+
+ var Acme = (function(acmeStore){
+  $(document).ready(function (){
+
+
+  var personalFireworks =" ";
+  var professionalFireworks = " ";
+  var noveltyFireworks = " ";
+
+acmeStore.showFireworks = function(products) {
+  console.log('showFireworks starting', products)
+  for(key in products){
```

Comment:
Good job using a for-in loop. For your multiple if-elses you could have also used a switch statement. If you really want a challenge, you could try to construct your html in such a way that it can accommodate any number of types (not just 3).